tests.py

```diff
@@ -1,46 +1,55 @@
 import graphene
 
-from .graphene_validator.decorators import validated
-from .graphene_validator.errors import (
+from graphene_validator.decorators import validated
+from graphene_validator.errors import (
     EmptyString,
     InvalidEmailFormat,
     LengthNotInRange,
     NegativeValue,
     NotInRange,
-    ValidationError,
+    SingleValidationError,
 )
 
 # Some dummy errors
 
 
-class NameEqualsAge(ValidationError):
-    pass
+class NameEqualsAge(SingleValidationError):
+    def __init__(self, path):
+        self.path = path
+
+    @property
+    def error_details(self):
```

Comment:
I'm a bit unclear on the use of this `error_details`. Why do we need to return a list/set of errors for each validation error, if each validation error only refers to an individual field? Also the idea behind custom errors was to make it optional to call `super` and just focus on the business logic validation (as DRF does).